extras/boost_test/include/rapidcheck/boost_test.h

```diff
@@ -8,14 +8,18 @@ namespace rc {
 namespace detail {
 
 template <typename Testable>
-void checkBoostTest(const std::string &description, Testable &&testable) {
+void checkBoostTest(Testable &&testable) {
   const auto &testCase = boost::unit_test::framework::current_test_case();
   TestMetadata metadata;
   metadata.id = testCase.full_name();
   metadata.description = testCase.p_name;
 
   const auto result = checkTestable(std::forward<Testable>(testable), metadata);
 
+  //Without this boost.test will complain about the test case having no assertions when the
```

Comment:
Formatting is incorrect, should have space after `//`.

Reply:
Fixed